rca/images/forms.py

```diff
@@ -0,0 +1,20 @@
+from django import forms
+from wagtail.images.forms import BaseImageForm
+
+
+class RCAImageForm(BaseImageForm):
+    """Custom image form with rights confirmation checkbox."""
+
+    rights_confirmed = forms.BooleanField(
+        label="I confirm I have the rights to use the image I am uploading",
+        help_text="Please confirm that you have the rights to use this image (owned, licensed, or free to use).",
+        required=True,
+        widget=forms.CheckboxInput(attrs={"class": "checkbox-input"}),
+    )
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Don't think this one's needed - the field should be added to the bottom of the form without this.